nano/templates/code_mirror.tmpl

```diff
@@ -0,0 +1,14 @@
+<h1>HELLO</h1>
```

Comment:
oh right that template, it was for testing the nanoUI elements
didn't remove it for anyone that every decided to fuck with nanoUI and codemirror